tf2onnx/onnx_opset/math.py

```diff
@@ -54,62 +54,76 @@ def version_9(cls, ctx, node, **kwargs):
         pass
 
 
+def make_minmax_op(ctx, op_type, inputs, outputs,
```

Comment:
the name looks confusing, maybe "make_min_or_max_op" or add a comment?

Reply:
sure